apps/ledger-live-desktop/tests/specs/accounts/delegate.smoke.spec.ts

```diff
@@ -0,0 +1,65 @@
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+import { AccountsPage } from "../../models/AccountsPage";
+import { Layout } from "../../models/Layout";
+import { AccountPage } from "../../models/AccountPage";
+import { Modal } from "../../models/Modal";
+
+test.use({ userdata: "accountCosmos" });
+
+test.beforeEach(async ({ page }) => {
+  const layout = new Layout(page);
+  await layout.goToAccounts();
+  const accountsPage = new AccountsPage(page);
+  await accountsPage.navigateToAccountByName("Cosmos 1");
+});
+
+test.describe("Delegate flow", async () => {
+  test("Delegate flow using max amount", async ({ page }) => {
+    const accountPage = new AccountPage(page);
+    const modalPage = new Modal(page);
+
+    await test.step("start the cosmos delegate flow", async () => {
+      await accountPage.startCosmosStakingFlow();
+      await expect.soft(page).toHaveScreenshot(`earn-reward-pre-flow-page.png`);
+    });
+
+    await test.step("Check Ledger is the provider by default", async () => {
+      await modalPage.continue();
+      const defaultprovider = await modalPage.getTitleProvider();
+      expect(defaultprovider).toEqual("Ledger");
+    });
+
+    await test.step("Toggle max amount to be filled in the ammount field", async () => {
+      await modalPage.continueDelegate();
+      await modalPage.toggleMaxAmount();
+      const availableMaxAmount = await modalPage.getSpendableBannerValue();
+      const filledMaxAmount = await modalPage.getCryptoAmount();
+      expect(filledMaxAmount).toEqual(availableMaxAmount);
+      await expect.soft(page).toHaveScreenshot(`staking-max-amount-page.png`);
+    });
+  });
+
+  test("The user search and select a provider", async ({ page }) => {
+    const accountPage = new AccountPage(page);
+    const modalPage = new Modal(page);
+
+    await test.step("start the cosmos delegate flow", async () => {
+      await accountPage.startCosmosStakingFlow();
+      await expect.soft(page).toHaveScreenshot(`earn-reward-pre-flow-page.png`);
```

Comment:
I'd recommend just doing a screenshot of the actual element here, not the full page